test/unit/services/services_spec.js

```diff
@@ -0,0 +1,121 @@
+"use strict";
+
+var Instance = require("../../../lib/services/services-db");
```

Comment:
This is an example of where naming conventions are important. In previous files, `Instance` has referred to the Instance model constructor, and here it refers to the Instance management service. Naming things carefully and being consistent is important to maintaining code in the future. It's worth stepping back and thinking about good, intuitive names for classes and services!